desci-server/src/services/Attestation.ts

```diff
@@ -506,6 +514,11 @@ export class AttestationService {
 
     if (duplicate) throw new DuplicateReactionError();
 
+    const attestation = await this.findAttestationById(claim.attestationId);
+    if (attestation.protected) {
+      await this.checkUserIsMember(authorId, claim.desciCommunityId);
+    }
```

Comment:
IMO, this is a bit confusing if you don't actively read `checkUserIsMember` first and figure out that it'll throw on failed check. Looks a bit like we check if they are a user, then just continue execution regardless :thinking: 

Personally, I'd actively check that return here, mainly because a refactoring of `checkUserIsMember` would be a type error, while the current implementation would silently ignore a return value.

Up to you how you wanna do it :)

Reply:
Good catch, I was supposed to clean that up after the test

Reply:
Refactored it to throw the error and renamed it to reflect the intent. The error thrown is handled properly when propagated to the errorHandler middleware.